frontend/config/chains.ts

```diff
@@ -16,8 +16,11 @@ export type ChainConfig = {
   evmChainId: number;
   middlewareChain: MiddlewareChainId;
   rpc: HttpUrl;
+  /**
+   * Least amount of native token required to create a Safe
```

Comment:
maybe we can have a todo saying that the values also hardcoded on the middleware now and we should pull the requirements endpoint to get them?